thesheriff/domain/bandido/puntuacion.py

```diff
@@ -1,9 +1,30 @@
 class Puntuacion:
-    def __init__(self, cantidadComida, calidadComida, servicio, precio):
-        self.cantidadComida = cantidadComida
-        self.calidadComida = calidadComida
+    """Puntuacion implements the score type for Asaltos.
+    :param cantidad_comida: score for the amount of food
+    :type cantidad_comida: float.
+    :param calidad_comida: score for the quality of food
+    :type calidad_comida: float.
+    :param servicio: score for the service
+    :type cantidad_comida: float.
+    :param precio: score for the price
+    :type precio: float.
+    :return: Puntuacion.
+    """
+
+    def __init__(
+            self, cantidad_comida: float, calidad_comida: float,
+            servicio: float, precio: float
+    ):
+        self.cantidad_comida = cantidad_comida
+        self.calidad_comida = calidad_comida
         self.servicio = servicio
         self.precio = precio
 
-    def value(self):
-        return (self.cantidadComida + self.calidadComida + self.servicio + self.precio) / 4
+    def value(self) -> float:
+        """value calculates the average score for an Asalto.
+        :return: float.
+        """
+        return (
+            self.cantidad_comida + self.calidad_comida + self.servicio +
+            self.precio
+        ) / 4
```

Comment:
A constant maybe? Instead of "magic numbers"

Reply:
makes complete sense, tho this file was added in an earlier PR, let's create a card to refactor it, I only modified as I needed for #23